Dockerfile

```diff
@@ -195,6 +195,7 @@ COLOR_RESET=$'\e[0m'
 readonly COLOR_RESET
 
 : "${INSTALL_MYSQL_CLIENT:?Should be true or false}"
+: "${MYSQL_CLIENT_TYPE:-mysql}"
```

Comment:
I'm not sure about name of env/arg maybe it is better to rename it to `INSTALL_MYSQL_CLIENT_TYPE`

Reply:
Yes. It's better to keep consistent prefix.